src/common/aes/aes128_armv8.c

```diff
@@ -3,15 +3,29 @@
 #include <assert.h>
 #include <stdint.h>
 #include <stdio.h>
+#include <string.h>
 #include <oqs/common.h>
 
 #include <arm_neon.h>
 
 #define PQC_AES128_STATESIZE 88
 typedef struct {
-	uint64_t sk_exp[PQC_AES128_STATESIZE];
+	uint64_t sk_exp[22];
```

Comment:
Yes, I think it can be removed. 
The `PQC_AES128_STATESIZE 88` was strange because expanded AES128 keys are only 176 bytes (=22*8 bytes) and not 88*8=704 bytes. The existing code also didn't read/write beyond 176 bytes. I see @dstebila originally co-authored the `PQC_AES128_STATESIZE 88`, double-checking if I missed anything here..

Reply:
@dstebila @bhess Just bumping this for resolution as it's (currently) the only unresolved comment on the PR. It's not exclusively a MAYO issue but it would be nice to sort it out from a code hygiene point of view.

Reply:
Yup, I'm not sure where 88 comes from. I went back to the original code in SUPERCOP and it needs a 176-byte array, so 22 unit64_t's should suffice.

Reply:
Thanks @SWilson4 & @dstebila, removed the macro now.